Makefile

```diff
@@ -28,7 +28,11 @@ install: no_test
 	mkdir -p $(DESTDIR)/lib
 	cp -RP library/libmbedtls.*    $(DESTDIR)/lib
 	cp -RP library/libmbedx509.*   $(DESTDIR)/lib
+ifdef USE_CRYPTO_SUBMODULE
+	$(MAKE) -C crypto install
+else
```

Comment:
As far as I understand, `crypto/include/mbedtls/config.h` is only meant to be used when crypto is used on its own. Standalone crypto, at this point, is primarily a demonstrator for the PSA API, so it activates a different set of features from mbedtls. In particular, we want PSA to be present in the default crypto build, but not in the default mbedtls build.

We need to make sure that the build scripts don't mix them up and to make the documentation clear.

Reply:
We have tests to ensure the build scripts don't mix them up. The documentation for how to use the submodule is in the crypto/README.md file, but it needs updating to explain how config.h is handled.

Reply:
Added a new commit for handling `make install` with submodule properly.

Reply:
For MbedOS 5.11 we will use only the mbed TLS config file - `include\mbedtls\config.h`. When the submodule is used on it's own, it will use its own `config.h` file.

In Mbed OS 5.12, we'll swap to two config.h files - one for TLS and X.509 in `include\mbedtls\config.h` and a separate one for Crypto.